include/mbgl/ios/MGLAnnotationImage.h

```diff
@@ -13,12 +13,12 @@ NS_ASSUME_NONNULL_BEGIN
 *   @param image The image to be displayed for the annotation.
 *   @param reuseIdentifier The string that identifies that this annotation image is reusable.
 *   @return The initialized annotation image object or `nil` if there was a problem initializing the object. */
-+ (instancetype)annotationImageWithImage:(UIImage *)image reuseIdentifier:(NSString *)reuseIdentifier;
++ (instancetype)annotationImageWithImage:(UIImage *)image reuseIdentifier:(nullable NSString *)reuseIdentifier;
```

Comment:
MGLMapView currently assumes that the reuse identifier is non-null. Is there a need for it to be nullable?